index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Project1</title>
+		<script src="https://kit.fontawesome.com/62fc5eef3c.js" crossorigin="anonymous"></script>
+	</head>
+	<body>
+		<!--Navbar -->
+		<header>
+			<nav>
+				<div class="nav-btns-left">
+					<p><a class="link" href="#about">About</a></p>
+					<p><a class="link" href="#portfolio">Portfolio</a></p>
+				</div>
+				<div class="nav-btn-right">
+						<p><a class="link" href="#contact">Contact</a></p>
+				</div>
+			</nav>
+		</header>
+	<!-- Main -->
+	 <main>
+		<div class="imgs-container">
+			<img src="kid.jpg" alt="kid statue">
+			<img src="london.jpg" alt="london eye image">
+			<img src="girl.jpg" alt="girl relaxing">
+			<img src="statue.jpg" alt="image of a statue">
+		</div>
+		<section class="hello-section">
+			<h1>Hello there!</h1>
+			<h2>Welcome to Cosmin's portfolio</h2>
```

Comment:
for the purposes of search engines, h1 is probably Welcome to Cosmin's Portfolio, because that is the true title of the page. so you should make the h1 into p, and make h2 into h1, and then use CSS to style them to look however you want